update-imported-tutorials.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
```

Comment:
Will add. I looked at update-imported-docs.sh, it wasn't there so I did not add initially.